src/app/components/single-image/single-image.component.scss

```diff
@@ -22,6 +22,10 @@ main {
     object-fit: cover;
 }
 
+button:focus {
+    outline: none;
+}
+
```

Comment:
Can we not move this button:focus styling to the app style sheet or the global style sheet since we declared in another file as well?

Reply:
i tried, it didnt work